exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/FileSelection.java

```diff
@@ -402,4 +402,18 @@ public String getCacheFileRoot() {
     return cacheFileRoot;
   }
 
+  @Override
+  public String toString() {
+    final StringBuilder sb = new StringBuilder();
+    sb.append("root=" + this.selectionRoot);
+
+    sb.append("files=[");
+    for (final String file : this.files) {
+      sb.append(file);
```

Comment:
Is this method only used internally or would it show up in the Explain output ?  If latter, you may add a comma or space after each file name.